biomage/stage/stage.py

```diff
@@ -245,6 +245,17 @@ def get_sandbox_id(templates, manifests, org, auto=False):
         return sandbox_id
 
 
+def get_pr_number(deployments, repo_name):
```

Comment:
this breaks when one just types `biomage stage`, i.e. when no PR number is specified. We are not guaranteed to have a PR number. Can you adapt the code to this case?

Reply:
No, it doesn't break. If there is no PR number, this function returns `None`. The `if` statement on Line 429 then return an empty string :

    "pipeline-pr": get_pr_number(deployments, "pipeline") or "",

I've just ran this:

<img width="1025" alt="image" src="https://user-images.githubusercontent.com/2862362/223363533-7dfc5184-10f2-4e17-a8af-a9954044aa6d.png">